src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -103,6 +103,9 @@ class LayoutLMv2FeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionM
         ocr_lang (`Optional[str]`, *optional*):
             The language, specified by its ISO code, to be used by the Tesseract OCR engine. By default, English is
             used.
+        tesseract_config (`Optional[str]`, *optional*):
```

Comment:
```suggestion
        tesseract_config (`str`, *optional*):
```
You can actually also remove "Optional" from the `ocr_lang` argument above (can't comment there)